base/typography.scss

```diff
@@ -18,27 +18,34 @@ h5,
 h6 {
   font-family: $header-font-family;
   line-height: $header-line-height;
+  font-weight: 600;
+  margin: 0;
 }
 
 h4,
 h5,
 h6 {
   font-weight: 400;
-  letter-spacing: 1px;
-  text-transform: uppercase;
 }
 
 h1 {
   font-size: $h1-font-size;
-  font-weight: 900;
+  &.dark {
+    font-size: $h1-dark-font-size;
+    color: $medium-gray
+  }
 }
 
 h2 {
   font-size: $h2-font-size;
+  color: $tws-orange;
 }
 
 h3 {
   font-size: $h3-font-size;
+  &.caps {
```

Comment:
Not part of the base typography. We are trying to get rid of all of these indecipherable class names. If you want this to be an option for all headers, create a components/_headers.scss file and make them all %extend a block like this.